documentation/modules/exploit/windows/http/northstar_c2_xss_to_agent_rce.md

```diff
@@ -0,0 +1,149 @@
UYmupx6GOJIAmtCNp2unolT9d4FEZGxyHfnCfs15KZvuqOUo6Q9fv5Sik6tbjLx/qHvenTnXr56/sC9jteVH/MDAy2ioUpENLBEJso3aNkz1oWYwy7v/ol/YWNUskxSrFGPmSjXwqlsJmb5GHIzZiUzkYimOisTX9091q4UQBUhyMLOTuUVOY//sVVHJj4Om814LZODMiDmcjhP/CVO99o2VImIBkYy0ZkQKPi22/ptFok225vCTmgUtur2q3c9E8PIojETiWiuRTLR/hpfRZl0Tm0/6COsVdfvzvehovvBMhNbN+APMQ1VIqKBMzLx2VPNIMmm4S8Iqvirgf4H8+uJlV8WbNTTGeT8cq3BgcV1VfMtEBHlztonyj4utPj7vI1sSl9+dWT963CjblNVvib71zYTHbn04FhmIhFNtsx35yu3g7dARJfBTCQiSpiJREQJM5GIKGEmEhEl45no3Hqm3Pr9E9HFMBPP1/ujkWO9RHRdDp+J5i8Ftv4ajP7h8PwvL5Z/jHxKJhZ/oBLwxzzzv6yNH82f1ky3Vy1C8Yc083smojHHzkSXPtlfE2wmSBlbWmnHaFU9EzXIBn85x/5JdV9vF5NhBDMWiWbgd2cYy7hiM2jTzb7GX/32rfxL2UgoNISain85J7zAbdgNY1QtJtl9Zm9KDgwtvU1T9C1FbSzGv6FEdAjMRJAs62ypsgGyWStTDxEjifPq7ungL2Xb1wUTQzEu43dkG6DVYtLIRHdUlndhQDhVuNvmTRb/EiXR7jETA9ku1VNGmQ2aZI0GqAmRynmacVPSLVvaxDkxAe1pq0WwmSivy41eiLlQ9FdEb3hH4Sg5edyHhrvit286DmZijJhGVCUxbopoG26spEllYibKye2BhoZgNS6zYnYe11IgmhvTejhD2idKvGo028ZMpONgJtqN1Qh8fb6LUSIqSWd+IW9SJqbxEknDXy6s1ivF+tuxG9u0T9RUDU3TM5wzz1miAzl4JlYTpJ2SaQ+V9l826Vz0IE1SsmQ5mPLIyjaqMt6PSWfDgT7UqkWjk4lyXRmA81dupvP2iQ5gUiY6mgVoqN6I1s2HUMi6wvfHTihI1+ivGIaG7VuWiWlrZs4g0RaTDnflDzdnS9lXLUatm9c7d4c8dHlt8zG2eFSrTrR/h87Eg8s2rRLWw+/sRAfDTDysfANb/fVKouNhJh5Z/h2ZgUjETCQispiJRETJITLRud07J6JLOkomEhFNwUwkIkqmZqKzeCy+QfNh7ohoHRfNRDzWtChMLhEtYfVMxINL68OME9EZVsxEPKl0WZh9IjrJKpmIp5O2g5UgopmWz0Q8lBPgAJoDczcNjiGiyRbORDyLDRhEi8LkNmAQEU2zZCbiKRxAN60Jc12DEUQ0wWKZiOcvhz66FMz7ALqJaMy8p6UTi3j4DHTQxWEBDHQQ0ZhlMhFPnoEO2giWQaFKRGNWyURUaVNYDIUqEXUxE3cLi6FQJaIuZuJuYTEUqkTUxUzcLSyGQpWIumY/KsNAdPDYKVRpU1gMhSoRdS3zqOCxU6jSprAYClUi6jpgJv759S+/9/o3v8dP+4XFUKgSUdeVZ+I/X//mez7CfvkLFFCJP05RHNLOxH/9PlwrtE/+GUonXC7391+kc8aL2iIutDwshkKViLpuJRPPCalpmSiBiLo7RMafl4mf/NSf88M/px/du3CBiIq+u7+Hn5aGxVCoElHXLWTib0KyIJhqAYf209f/CjVJt09k0/dT5I40H3maiR9qUSJJ9m4xvzzNLGkSl8MtXnY5m55yb908ldBcZ6uIxVCoElHXTWTi70122Ey0oSP1EIvxKzD2X40Y9b3mDPEobBVFfqwbg15TLy+nKiE7ILnMfSLR1biRTEzhZcJIQidusmJuSkilaKtlYuz1qZRvMKUNs68QDywvp4ovzkNy/8MDF4LFUKgSUdetZGJMwD+nkCq+eMaIlJBKYTQ5E0VMxnJbGsRv3L6ZTBxmX3+fiKvU0nYhWAyFKhF13U4mxnjyqRSipLpPdN9Dz8xEJ8VZfmy8hFPsE4fZ10s98519NVgMhSoRdd1QJsaUiWli082E1zCksuBrZKLLuxivvjjIPsdnoryWUOtmoiPniV0uav0lLhGIDhZDoUpEXTeViY6kTAyUlJKDog2pOMyfqpGJSFVt8fc9hsdK+9DtJccy0ZGtpTS5qK3Y+tKwGApVIuq68kyk02ExFKpE1MVM3C0shkKViLqYibuFxVCoElEXM3G3sBgKVSLqYibuFhZDoUpEXczE3cJiKFSJqIuZuFtYDIUqEXUxE3cLi6FQJaIuZuJuYTEUqkTUdeWZmP91Y+9CfzFuB7AYClUi6mIm7hYWQ6FKRF03non468/2rxvbv7as/3aD/JXkyj+FvWdYDIUqEXXdcibafyvsk1+Yf8fBJmZ4Hf8dh3X+RevrhMVQqBJR1+1nov1HZYp/okYGxH9RcZ1/fuZqYTEUqkTUdcuZ6MTvzjYlixYzsfXPee0UFkOhSkRdN5GJ+suCXsjEYscn/yKh+xJtv01bzERmItE0V56JmncxzmL8yWt5kaIw30VGzERmItE0V5+JTvGNOP0+iewiQ0t5J7EYW8hHZiIzkWiaW8hEOgkWQ6FKRF3MxN3CYihUiaiLmbhbWAyFKhF1MRN3C4uhUCWiLmbibmExFKpE1MVM3C0shkKViLqYibuFxVCoElEXM3GfsBIGOoioa5VMdNBBG8EyKFSJaMxiTwsePgMddFmY/Rz6iGjMipko0E3rw4wPoJuIJljygcEj2IBBtAJMcQMGEdEECz8weAq7MJTOg9kcg9FENM3yzwyeRdoa1oOI5ljrycFzSRvBMhDRTOs+PHhA6YIw9UR0kgs9QnheaTWYaCI6zwbPEh5iOhsmlIiWw+eKiChhJhIRJcxEIqKEmUhElDATiYgSZiIRUcJMJCJKmIlERAkzkYgoYSYSEanXr/8fOKzy3Mx0GKYAAAAASUVORK5CYII=)
```

Comment:
Our docs site and github supports mermaid diagrams: https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/ - which make it easier to allow other contributors to change this diagram in the future if need be

I use this editor a lot: https://mermaid.live/

Reply:
I had never seen/heard of this before, pretty awesome, thanks for the info!